src/testCases/testCases.controller.ts

```diff
@@ -3,26 +3,30 @@ import { ControllerUtils } from 'src/utils/controller/utils';
 import { TestCaseViewModel } from './testCase.viewModel';
 import { TestCasesService } from './testCases.service';
 
-@Controller('test_cases')
+@Controller('repos/:owner/:repo/test_cases')
```

Comment:
This may have just been one place I missed to update.

Reply:
Oh I just realised I read this backwards. I thought the diff was a suggested change.

Yes while you're correct I think this makes more sense from a user standpoint as it keeps the url prettier.

For example Github issues and prs share the same id pool. With the url `https://github.com/ably-labs/test-observability-server/pull/22` you can actually change the pull segment to issue and it will work the same. It doesn't matter it's just there as visual aid.

Reply:
OK, I don't have a strong opinion either way, so happy to keep it how you've done it.